tools/IceRpc.Slice.Tools/README.md

```diff
@@ -68,10 +67,10 @@ your NuGet package to easily retrieve these Slice files.
 You can also define `SliceDirectory` in this NuGet package so that any project that references this package
 automatically sees these Slice files when compiling. For example:
 
+### `buildTransitive/<PackageName>.props`
```

Comment:
Are we confident that NuGets markdown rendering supports inline code blocks in headings? I think ive seen some markdown renders in the past that dont do it.

Reply:
I will check, you can preview the README when uploading a package

Reply:
I would do instead just like above:

For example, if you pack Slice files into the `slice` directory of your NuGet package, you want to include the following `ItemGroup` in the `buildTransitive/<PackageName>.props` you pack into this package:
```...
```